src/main/content/_layouts/intro.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: doc
+type: intro
```

Comment:
I don't see where the type attribute is used.

Reply:
What would be its future use?

Reply:
Checking the `page.type` for intro type pages, rather than checking `page.url` for containing `/docs/intro`

Reply:
I would remove it unless 100% certain it will be used in the future, because it could become dead code if never used or looked at again.